### ISSUE TITLE
user과 group의 formatting에 관한 질문

### ISSUE THREAD

**Issue by PJH6029** [SNU-ARC/2024_spring_sysprog_Lab2#13]
reference의 dirtree와 제 dirtree를 비교하는 과정에서 생긴 이슈입니다.

/proc 을 대상으로 실행하는 과정에서 reference dirtree의 formatting이 깨지는 것을 발견하였습니다.
(제가 짠 코드에서도 마찬가지로 깨졌습니다)
아마 username과 group이 8자 이상이라 생긴 현상 같은데,
스펙에 명시된대로 해당 케이스는 채점하지 않는 것일까요?

실행 명령어: `./reference/dirtree /proc -v`

![image](https://github.com/SNU-ARC/2024_spring_sysprog_Lab2/assets/46991571/955cf798-97b3-4a86-8847-3d6b82cfde5d)

**Comment by Youngook-Song** [comment]
네, Output formatting Table에서 ignore로 표기된 항목은 따로 처리를 하지 않으셔도 됩니다.
